tests/test_entrypoint.py

```diff
@@ -148,16 +151,16 @@ def test_entrypoint_add_parameters():
     try:
         args = parser.parse_args()
         for arg in vars(args):
-            assert arg in check_result and check_result[arg] == str(getattr(args, arg))
-    except SystemExit as sysexit:
-        assert str(sysexit) != "0"
+            assert arg in check_result and check_result[arg] == getattr(args, arg)
+    except SystemExit:
+        assert False
```

Comment:
```suggestion
        assert False  # SystemExit exception was caught
```
Adding the comment to provide some context to the developer that sees the error (`assert False` is not very descriptive)